superset/cachekeys/api.py

```diff
@@ -0,0 +1,98 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think the route `/api/v1/cachekey/invalidate` is not ideal. I'm struggling to find something good though

Reply:
agree, I had the same struggles, definitely open to suggestions. 
/api/v1/cache/invalidate could be slightly better

Reply:
How about `DELETE /api/v1/cachekey/<cachekey>`?

Reply:
it would be too much struggle to fetch cache keys though the API and then issue deletes for those.
I think we could expose GET / DELETE later once those use cases will arise